src/swigpytests.cc

```diff
@@ -31,6 +32,47 @@ swigpytests::cc_trust_data::cc_trust_data() {
 
 swigpytests::cc_trust_data::~cc_trust_data() {}
 
+bool swigpytests::cc_trust_data::initialize_simulated_enclave(
```

Comment:
Method copy-cloned and added to SWIG-unit-test exerciser, to verify that the SWIG mapping [etc.] is working right.